src/components/RadioButtonGroup/radioButton.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import RenderIf from '../RenderIf';
+import { uniqueId } from '../../libs/utils';
+
+export default class RadioButton extends Component {
+    constructor(props) {
+        super(props);
+        this.radioId = uniqueId('radiobutton');
+    }
+
+    getContainerClassNames() {
+        const { isChecked, disabled } = this.props;
+        return classnames(
+            'rainbow-radio-button-group_radio',
+            {
+                'rainbow-radio-button-group_radio--checked': isChecked,
+            },
+            {
+                'rainbow-radio-button-group_radio--disabled': disabled,
+            },
+        );
+    }
+
+    getLabelClassNames() {
+        const { disabled } = this.props;
+        return classnames('rainbow-radio-button-group_radio-label', {
+            'rainbow-radio-button-group_radio-label--disabled': disabled,
+        });
+    }
+
+    render() {
+        const {
+            label,
+            ariaDescribedby,
+            onChange,
+            value,
+            disabled,
+            isChecked,
+            name,
+            required,
+        } = this.props;
+
+        return (
+            <span className={this.getContainerClassNames()}>
+                <input
+                    ref={this.ref}
+                    type="radio"
+                    required={required}
+                    id={this.radioId}
+                    name={name}
+                    value={value}
+                    checked={isChecked}
+                    aria-describedby={ariaDescribedby}
+                    onChange={onChange}
+                    disabled={disabled}
+                />
+
+                <label
+                    className="rainbow-radio-button-group_radio-label-container"
+                    htmlFor={this.radioId}
+                >
+                    <RenderIf isTrue={!!label}>
```

Comment:
you can remove the RenderIf since the label is required for accessibility, in RadioGroup we should remove it too